Source/Flow/Public/Interfaces/FlowExternalExecutableInterface.h

```diff
@@ -0,0 +1,33 @@
+// Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
+
+#pragma once
+
+#include "UObject/Interface.h"
+#include "UObject/ScriptInterface.h"
+
+#include "FlowExternalExecutableInterface.generated.h"
+
+// Forward Declarations
+class IFlowNativeExecutableInterface;
+
+// Implemented by external objects that can execute within a flow graph
+// via a FlowNode or FlowNodeAddOn proxy
+UINTERFACE(MinimalAPI, Blueprintable, DisplayName = "Flow External Executable Interface")
+class UFlowExternalExecutableInterface : public UInterface
+{
+	GENERATED_BODY()
+};
+
+class FLOW_API IFlowExternalExecutableInterface
```

Comment:
This interface can be implemented by "external" (ie, non-flow node/addon) objects to become executable via a proxy node or addon in a flow graph.

See the ExecuteComponent flow node for example usage.  If the component implements this interface and the Core flow interface, it can be executed by that node.